java/src/main/java/org/rocksdb/TableProperties.java

```diff
@@ -292,6 +292,18 @@ public long getFastCompressionEstimatedDataSize() {
     return fastCompressionEstimatedDataSize;
   }
 
+  /**
+   * Get the  offset of the value of the property
+   * "external sst file global seqno" in the file
+   * if the property exists.
+   *
+   * @return the offset of the property
+   * "external sst file global seqno" or 0 if not exists
+   */
+  public Long getExternalSstFileGlobalSeqnoOffset() {
```

Comment:
TODO:
Delete this since we don't want internal details like block offset exposed.

Reply:
Done